lib/transloadit/Transloadit.php

```diff
@@ -135,4 +135,93 @@ public function cancelAssembly($assembly_id) {
       return $response;
     }
   }
+
+  /**
+   * Generates a signed URL for Transloadit's Smart CDN
+   * https://transloadit.com/services/content-delivery/
+   *
+   * @param string $workspaceSlug The workspace slug
+   * @param string $templateSlug The template slug
+   * @param string $inputField The input field (optional)
+   * @param array $params Additional parameters (optional)
+   * @param array $signProps Array containing authKey, authSecret, and expireAtMs
+   * @return string The signed URL
+   */
+  public function signedSmartCDNUrl(
+      string $workspaceSlug,
+      string $templateSlug,
+      string $inputField = '',
+      array $params = [],
+      array $signProps = []
+  ): string {
+    // Validate required fields
+    if (!$workspaceSlug) {
+      throw new \InvalidArgumentException('workspace is required');
+    }
+    if (!$templateSlug) {
+      throw new \InvalidArgumentException('template is required');
+    }
+    if (!$inputField) {
+      throw new \InvalidArgumentException('input is required');
+    }
+
+    // Add auth parameters
+    $queryParams = [];
+
+    // Process params to match Node.js behavior
+    foreach ($params as $key => $value) {
+      if (is_array($value)) {
+        foreach ($value as $val) {
+          if ($val !== null && $val !== '') {
```

Comment:
Same as in the Ruby SDK PR. Can we allow empty string values?

Reply:
Do you want the empty parameter to show up? I don't think the node implementation did that at the time of writing?

For instance: 

```
    // Test with empty param string
    $url = $transloadit->signedSmartCDNUrl('workspace', 'template', 'file.jpg', ['width' => '', 'height' => '200']);
    $this->assertStringNotContainsString('width=', $url, 'Empty parameter should be excluded from URL');
```

or

 ```
    // Test with empty param string
    $url = $transloadit->signedSmartCDNUrl('workspace', 'template', 'file.jpg', ['width' => '', 'height' => '200']);
    $this->assertStringContainsString('width=', $url, 'Empty parameter should be excluded from URL');
```

Reply:
Okay, let's also fix this on the node side then, currently these SDKs are complaining about that bit

Reply:
Or maybe it was a bad port by me 🤔

Reply:
The Node SDK PR already supports empty values in my tests.